rpm/native_db.go

```diff
@@ -299,3 +426,106 @@ func constructHint(b *strings.Builder, info *Info) string {
 	}
 	return b.String()
 }
+
+func findDBs(ctx context.Context, out *[]foundDB, sys fs.FS) fs.WalkDirFunc {
+	return func(p string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if d.IsDir() {
+			return nil
+		}
+
+		dir, n := path.Split(p)
+		dir = path.Clean(dir)
+		switch n {
+		case `Packages`:
+			f, err := sys.Open(p)
+			if err != nil {
+				return err
+			}
+			ok := bdb.CheckMagic(ctx, f)
+			f.Close()
+			if !ok {
+				return nil
+			}
+			*out = append(*out, foundDB{
+				Path: dir,
+				Kind: kindBDB,
+			})
+		case `rpmdb.sqlite`:
```

Comment:
I know it's copied, but the others check using a magic number. We don't bother with that here? I guess trusting the file extension `sqlite` is good enough

Reply:
I don't know about this, I feel like I've heard a reason when we were debugging prod errors but can't recall. @hdonnay any ideas?

Reply:
Yeah, this just hasn't come up as an issue, whereas relying on file names for the others has.